components/merkledb/examples/manual_migration.rs

```diff
@@ -0,0 +1,145 @@
+// Copyright 2020 The Exonum Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Shows how to migrate database data. The migration follows the following scenario:
```

Comment:
Nit (here and in other files): "follow the following" sounds a bit odd, I'd suggest something like "goes as following" or "is performed as following".